packages/remix/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@clerk/clerk-remix",
+  "version": "3.0.0-alpha.0",
+  "license": "MIT",
+  "description": "Clerk.dev SDK for Remix",
+  "keywords": [
+    "clerk",
+    "typescript",
+    "remix",
+    "auth",
+    "authentication",
+    "passwordless",
+    "session",
+    "jwt"
+  ],
+  "sideEffects": false,
```

Comment:
Instead of this, we can rename `ssr.js` to `ssr.server.js` - Remix will bundle the files correctly then.

The imports would become:
```tsx
import { getAuth } from "@clerk/clerk-remix/ssr.server"
```

Reply:
I think I prefer `"sideEffects": false` over the longer import.

If we keep `"sideEffects": false`, is it possible to bundle `rootAuthLoader`, `getAuth`, etc on the root instead of `/ssr`?

Reply:
Talked with @igneel64 , decided to go with the `ssr.server` convention instead, because its hard to predict how an external bundler will behave in the future - we don't want to risk any files not being bundled accidently from a change we didn't make